enterprise/backend/src/metabase_enterprise/dashboard_subscription_filters/pulse.clj

```diff
@@ -6,12 +6,10 @@
   "Enterprise way of getting dashboard filter parameters. Blends parameters from dashboard subscription and the
   dashboard itself."
   :feature :dashboard-subscription-filters
-  [pulse dashboard]
-  (let [pulse-params           (:parameters pulse)
-        dashboard-params       (:parameters dashboard)
-        pulse-params-by-id     (group-by :id pulse-params)
+  [dashboard-subscription-params dashboard-params]
```

Comment:
This is one of the examples of my quest to strip down the function to only receive what it needs. We need to strip it down to the minimum because we will soon rework the pulse schema, and we need to know what are the minimal required columns.